src/OpenTelemetry.Exporter.Geneva/Metrics/GenevaMetricExporter.cs

```diff
@@ -119,6 +123,9 @@ public GenevaMetricExporter(GenevaMetricExporterOptions options)
 
     public override ExportResult Export(in Batch<Metric> batch)
     {
+        string monitoringAccount = this.defaultMonitoringAccount;
```

Comment:
as an enhancement, consider making this optional. i.e users can avoid specifying account, ns in connectionstring.

Reply:
I'll do this in a follow up PR.